### PR TITLE
As requested on Twitter, here is an alternative to get the UDID witho…

### PR DESCRIPTION
**Pull request by ddouhine**
…ut installing additional tools)

Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #< insert number here >.

### CODE REVIEW

File: Document/0x06b-Basic-Security-Testing.md [hunk 1]
```diff
@@ -31,12 +31,11 @@ $ xcode-select --install
 
 The UDID is a 40-digit unique sequence of letters and numbers to identify an iOS device. You can find the [UDID of your iOS device via iTunes](http://www.iclarified.com/52179/how-to-find-your-iphones-udid "How to Find Your iPhone's UDID"), by selecting your device and clicking on "Serial Number" in the summary tab. When clicking on this you will iterate through different meta-data of the iOS device including its UDID.
 
-It is also possible to get the UDID via the command line, from a device attached via USB. Install `ideviceinstaller` via brew and use the command `idevice_id -l`:
+It is also possible to get the UDID via the command line, from a device attached via USB using `ioreg`:
```
Comment (cpholguera) [inline Document/0x06b-Basic-Security-Testing.md]:
```suggestion
On macOS, it is also possible to get the UDID of a device attached per USB via the command line,  using the [I/O Registry Explorer](https://developer.apple.com/library/archive/documentation/DeviceDrivers/Conceptual/IOKitFundamentals/TheRegistry/TheRegistry.html "I/O Registry Explorer") tool `ioreg`:
```

**Comment by commjoen** [comment]
Can we please have both? (E.g. add next to idevice_id command :-))

**Comment by cpholguera** [comment]
Is the other approach working for you? For me isn't working, or my installation is messed up. @ddouhine could you please test it? That'd be very helpful :)

**Comment by commjoen** [comment]
> Is the other approach working for you? For me isn't working, or my installation is messed up. @ddouhine could you please test it? That'd be very helpful :)

Both are working for me, that's why i rather keep both 👍

**Comment by cpholguera** [comment]
Thanks for testing @commjoen . Very nice, then yes, let's keep both :)

**Comment by ddouhine** [comment]
Both are working for me too:
```
$idevice_id -l
64655621de6ef5e56a874d63f1e1bdd14f7103b1
$ioreg -p IOUSB -l | grep "USB Serial"
  | |       "USB Serial Number" = "DLC648301LXG2KQAM"
  |   |     "USB Serial Number" = "DLC632503AVG2WFAA"
  |         "USB Serial Number" = "64655621de6ef5e56a874d63f1e1bdd14f7103b1"
```
And here is an additional way to get the UDID (credit: https://twitter.com/samykamkar):
```
$system_profiler SPUSBDataType | sed -n -e '/iPad/,/Serial/p;/iPhone/,/Serial/p;/iPod/,/Serial/p' | grep "Serial Number:"
2019-09-08 10:18:03.920 system_profiler[13251:1050356] SPUSBDevice: IOCreatePlugInInterfaceForService failed 0xe00002be
              Serial Number: 64655621de6ef5e56a874d63f1e1bdd14f7103b1
```

**Comment by commjoen** [comment]
Can you please put all 3 in there :-)?

**Comment by commjoen** [comment]
Thanks!

**Pull request merged**